packages/core/test/fixtures/transaction.ts

```diff
@@ -597,6 +647,7 @@ export const validTransactions = [
                     script: '76a914851a33a5ef0d4279bd5854949174e2c65b1d450088ac',
                 },
             ],
+            inPubkeys: [null],
```

Comment:
This is not right. It should not be null.
Pub key is `038de63cf582d058a399a176825c045672d5ff8ea25b64d28d4375dcdb14c02b2b`

```suggestion
            inPubkeys: ['038de63cf582d058a399a176825c045672d5ff8ea25b64d28d4375dcdb14c02b2b'],
```